exercises/E-conditionals/exercise.js

```diff
@@ -6,9 +6,17 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-let name = "Daniel";
+let myName = "Daniel";
 let danielsRole = "mentor";
-
+let message = "";
+if (myName === "Daniel") {
+  danielsRole === "mentor"
+    ? (message = "Hi my name is Daniel and I'm a mentor.")
```

Comment:
This looks good - but maybe you could use the myName variable instead of repeating Daniel's name here.